docs/guides/user-containers/readme.md

```diff
@@ -6,7 +6,55 @@ applications to be run on Rabbit nodes with access to NNF ephemeral and persiste
 !!! note
 
     The following is a limited look at User Containers.  More content will be
-    provided after the RFC has been finalized.
+    provided after the RFC has been finalized. This is a work in progress.
+
+## Environment Variables
+
+Two sets of environment variables are available with container workflows: Container and Compute Node. The former are the variables that are available inside the user containers. The latter are the variables that are provided back to the DWS workflow, which in turn are collected by Flux and provided to compute nodes. See the Flux documentation for more details.
+
+### Container Environment Variables
+
+These variables are provided for use inside the container. They can be used as part of the
+container command in the NNF Container Profile or within the container itself.
+
+#### Container Hostnames and Domains
+
+Containers can contact one another via Kubernetes cluster networking. This functionality is provided by DNS. Environment variables are provided that allow a user to be able to piece together the FQDN so that the other containers can be contacted.
+
+This example demonstrates an MPI container workflow, with two worker nodes. Two worker nodes means two containers running on two NNF nodes.
```

Comment:
"Two worker nodes means two containers running on to NNF nodes."
So these are generic k8s worker nodes, explicitly not Rabbits? Or are they something else?

Reply:
This should say two worker pods, not nodes.